modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -29,7 +29,7 @@ To dynamically create a volume snapshot:
 .volumesnapshotclass.yaml
 [source,yaml]
 ----
-apiVersion: snapshot.storage.k8s.io
+apiVersion: snapshot.storage.k8s.io/v1beta1
```

Comment:
Is this correct back to 4.7?

Reply:
I think yes.
@bertinatto could you kindly confirm from 4.4, it is v1beta1 version?

Reply:
Talked to @huffmanca and it should be `v1beta1`, yes. We're moving to `v1` in OCP 4.7: https://github.com/openshift/cluster-csi-snapshot-controller-operator/pull/69

Reply:
[] Open a separate PR to update to `v1` starting in OCP 4.7 docs off `update-v1-1906518` feature branch.

Reply:
Opened https://github.com/openshift/openshift-docs/pull/28289 to change to `v1` beginning in 4.7.

Reply:
I'm now going to need to open a separate PR against 4.4, 4.5, and 4.6 that changes instances in this section to `snapshot.storage.k8s.io/v1beta1`.

Reply:
4.5 update: https://github.com/openshift/openshift-docs/pull/28330
4.6 update: https://github.com/openshift/openshift-docs/pull/28331